### ISSUE TITLE
Is there a reason for returning non zero values when a functions succeeds?

### ISSUE THREAD

**Issue by codewitchio** [Zalodu/Quadcontrol-2#10]
Currently we've stuck to returning ``0`` upon success and ``-1`` upon failure, effectively returning ``TRUE`` when it fails and ``FALSE`` when it works. Is there a reason this is still in place after all troubleshooting at that level is done? I feel like switching to ``1`` for success and ``0`` for failure would make ``if`` clauses more readable. In my opinion, this doesn't look very logical and is hard to follow: ![image of code](http://i.imgur.com/4E2MVgh.png)

@MrLarssonJr

**Comment by Abendrill** [comment]
That is a very valid point.

The only counter argument I can think of is that I think that's how Unix do?

But I could agree to switch.

**Comment by codewitchio** [comment]
Let's keep it consistent, ``0`` means a success, anything else means some kind of fail. This is better because it allows us to use the return value to signal what went wrong where.
